modules/UI/videolayout/VideoContainer.js

```diff
@@ -426,14 +426,17 @@ export class VideoContainer extends LargeContainer {
         const { horizontalIndent, verticalIndent }
             = this.getVideoPosition(width, height, containerWidth, containerHeight);
 
+        const { isOpen } = APP.store.getState()['features/chat'];
+        const horizontalIndentLeft = isOpen ? 0 : horizontalIndent;
```

Comment:
`getVideoPosition` is centring the content. Why do you need to set `left` to `0`?

Reply:
Because left is set to a negative value here and thus a small portion of it is hidden behind the chat